content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx

```diff
@@ -0,0 +1,430 @@
+---
+title: 'Upgrade to Prisma 5'
+metaTitle: 'Upgrade to Prisma 5'
+metaDescription: 'Guides on how to upgrade to Prisma 5'
+tocDepth: 3
+toc: true
+---
+
+<TopBlock>
+
+Prisma 5.0.0 introduces a number of changes, including the usage of our new JSON Protocol, [which make Prisma faster by default](https://www.prisma.io/blog/prisma-and-serverless-73hbgKnZ6t#improved-startup-performance-by-9x)! A list of these changes can be found [in our release notes](https://github.com/prisma/prisma/releases/tag/5.0.0).
```

Comment:
Do we have a policy if we say "Prisma 5.0.0" or "Prisma version 5.0.0" or anything similar?

Reply:
I would advocate for `Prisma 5` and `Prisma version 5.0.0` but no strong preferences